### ISSUE TITLE
5.x: Ctrl+s in editor uses form submit instead of AJAX call

### ISSUE THREAD

**Issue by scolytus** [gollum/gollum#1356]
While playing with gollum and using `ctrl` + `s` by habit in the editor, I noticed that it actually works and saves the file. However, it simply submits the form, issuing a `POST` request to e.g. `/gollum/gollum/edit/Home.md`. The result is HTTP 200 and the page is saved, but the returned payload is just a string, I suppose a git commit ID.

I think I also identified the JS:

https://github.com/gollum/gollum/blob/c31c68200bf5345c1bcdad8fb5cdfee7b21ef7af/lib/gollum/public/gollum/javascript/editor/gollum.editor.js.erb#L87

Would be better to use the same JS event handler that posts it via Ajax.

Maybe this can also be fixed in #1338 

If not, maybe I can try to create a PR on my own, but I have a rather clumsy dev setup here at the moment, would take some time.

Versions:

```
Gollum 5.0.1b
Running on: x86_64-linux with Ruby version 2.5.3
Using:
gollum-grit_adapter 1.0.1
gollum-lib 5.0.a.4
With the following renderers:
kramdown 1.17.0
RedCloth 4.2.9
org-ruby 0.9.12
wikicloth 0.8.3
```

**Comment by dometto** [comment]
Thanks for the report! Will fix in #1338

**Comment by dometto** [comment]
Fixed in https://github.com/gollum/gollum/pull/1361
